docs/download.md

```diff
@@ -4,6 +4,205 @@ title: Download
 nav_order: 3
 ---
 
+
+## Download link and version matrix
+
+<table border=0 cellpadding=0 cellspacing=0 width=1442 style='border-collapse:
```

Comment:
Can we not use a markdown table here?

Reply:
@jlowe I can try to convert HTML table to a markdown table. Should I open a new PR and close this one?thx